2016-12-24 00:03:36,537 - knowledge_logger - main.py-handle-api- : 'c_task' 

2016-12-24 00:03:36,566 - knowledge_logger - main.py : Traceback (most recent call last):
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 202, in handle_api_request
    response_data = unbound_method(self, request_data, _db)
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 351, in handle_techno_report
    return controller.process_techno_report_request(request, db)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technoreportcontroller.py", line 86, in process_techno_report_request
    result = process_get_assigned_statutory_report_filters(db, user_id)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technoreportcontroller.py", line 264, in process_get_assigned_statutory_report_filters
    compliance_statutories = get_compliance_statutoy_for_statutorysetting_report(db, user_id)
  File "/home/usha/aparajitha-saas/Src-server/server/database/technoreport.py", line 78, in get_compliance_statutoy_for_statutorysetting_report
    return return_compliance_statutory(db, result[1])
  File "/home/usha/aparajitha-saas/Src-server/server/database/technoreport.py", line 87, in return_compliance_statutory
    st_cmp["c_task"], st_cmp["document_name"], st_cmp["statutory_name"]
KeyError: 'c_task'
 

2016-12-24 00:03:40,835 - knowledge_logger - main.py-handle-api- : sp_statutory_mapping_report_frequency- ()- 1305 (42000): PROCEDURE compfie_knowledge_updated.sp_statutory_mapping_report_frequency does not exist 

2016-12-24 00:03:40,836 - knowledge_logger - main.py : Traceback (most recent call last):
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 202, in handle_api_request
    response_data = unbound_method(self, request_data, _db)
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 343, in handle_knowledge_report
    return controller.process_knowledge_report_request(request, db)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/knowledgereportcontroller.py", line 48, in process_knowledge_report_request
    db, request_frame, user_id
  File "/home/usha/aparajitha-saas/Src-server/server/controller/knowledgereportcontroller.py", line 98, in process_get_statutory_mapping_filters
    compliance_frequency = get_compliance_frequency(db)
  File "/home/usha/aparajitha-saas/Src-server/server/database/general.py", line 420, in get_compliance_frequency
    "sp_statutory_mapping_report_frequency", ()
  File "/home/usha/aparajitha-saas/Src-server/server/dbase.py", line 783, in call_proc
    raise process_procedure_error(procedure_name, args, e)
RuntimeError: sp_statutory_mapping_report_frequency- ()- 1305 (42000): PROCEDURE compfie_knowledge_updated.sp_statutory_mapping_report_frequency does not exist
 

2016-12-24 00:30:15,602 - knowledge_logger - main.py-handle-api- : 'c_task' 

2016-12-24 00:30:15,602 - knowledge_logger - main.py : Traceback (most recent call last):
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 202, in handle_api_request
    response_data = unbound_method(self, request_data, _db)
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 351, in handle_techno_report
    return controller.process_techno_report_request(request, db)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technoreportcontroller.py", line 86, in process_techno_report_request
    result = process_get_assigned_statutory_report_filters(db, user_id)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technoreportcontroller.py", line 264, in process_get_assigned_statutory_report_filters
    compliance_statutories = get_compliance_statutoy_for_statutorysetting_report(db, user_id)
  File "/home/usha/aparajitha-saas/Src-server/server/database/technoreport.py", line 78, in get_compliance_statutoy_for_statutorysetting_report
    return return_compliance_statutory(db, result[1])
  File "/home/usha/aparajitha-saas/Src-server/server/database/technoreport.py", line 87, in return_compliance_statutory
    st_cmp["c_task"], st_cmp["document_name"], st_cmp["statutory_name"]
KeyError: 'c_task'
 

2016-12-24 00:58:31,189 - knowledge_logger - main.py-handle-api- : sp_get_user_category_id_by_userid- (5,)- 1305 (42000): PROCEDURE compfie_knowledge_updated.sp_get_user_category_id_by_userid does not exist 

2016-12-24 00:58:31,190 - knowledge_logger - main.py : Traceback (most recent call last):
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 202, in handle_api_request
    response_data = unbound_method(self, request_data, _db)
  File "/home/usha/aparajitha-saas/Src-server/server/main.py", line 315, in handle_techno
    return controller.process_techno_request(request, db)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technocontroller.py", line 149, in process_techno_request
    result = get_unassigned_units(db, session_user)
  File "/home/usha/aparajitha-saas/Src-server/server/controller/technomastercontroller.py", line 548, in get_unassigned_units
    user_category_id = get_user_category_id(db, session_user)
  File "/home/usha/aparajitha-saas/Src-server/server/database/technomaster.py", line 2271, in get_user_category_id
    result = db.call_proc("sp_get_user_category_id_by_userid", (int(session_user),))
  File "/home/usha/aparajitha-saas/Src-server/server/dbase.py", line 783, in call_proc
    raise process_procedure_error(procedure_name, args, e)
RuntimeError: sp_get_user_category_id_by_userid- (5,)- 1305 (42000): PROCEDURE compfie_knowledge_updated.sp_get_user_category_id_by_userid does not exist
 

